src/pages/test-report/cat-cpc/components/question-del-ex-radio-card/question-del-ex-radio-card.scss

```diff
@@ -0,0 +1,2 @@
+question-del-ex-radio-card {
```

Comment:
If this file doesn't contain any styling we can remove it entirely

Reply:
good point - removed